docs/api-docs/building storefronts/nextjs-commerce-guide.md

```diff
@@ -1,5 +1,16 @@
 # Next.js Commerce Quick Start Guide
 
+<div class="otp" id="no-index">
+
+### On this page
```

Comment:
Please update the names and capitalizations in the main body to match the corrections below.